Dockerfile.dist

```diff
@@ -17,9 +17,11 @@
 ## Build
 
 # This works fine but will produce an EC violation
-# FROM docker.io/library/golang:1.21 AS build
+# FROM docker.io/library/golang:1.22 AS build
 
-FROM registry.access.redhat.com/ubi9/go-toolset:1.21@sha256:ae17d73e70a966f39ef4dfca74241e3ca4374cd1198b02c30ea0748b8dcc83a6 AS build
+# Ideally, use the official image from Red Hat, but a 1.22 release does not yet exist.
+# FROM registry.access.redhat.com/ubi9/go-toolset:1.21@sha256:ae17d73e70a966f39ef4dfca74241e3ca4374cd1198b02c30ea0748b8dcc83a6 AS build
```

Comment:
I'd remove this it is pointing to 1.21...

```suggestion
```

Reply:
Fair. I re-wrote the comment and remove that specific image ref.